README.md

```diff
@@ -48,76 +31,68 @@ You need to have the following installed:
 If you want to benchmark Java, you will also need:
   * A Java SDK 7 (`openjdk-7-jdk` package in Debian)
 
-Note that to use pinning on Linux, `cset shield` must be in a working state.
-Some Linux distributions have been known to ship with this functionality
-broken. See the `cset` tutorial for information on how to test `cset shield`:
-https://rt.wiki.kernel.org/index.php/Cpuset_Management_Utility/tutorial
+You will need either `sudo` (on Linux) or `doas` (OpenBSD) to be setup.
 
-### Kernel arguments
 
-If you are using a Linux system, you will need to set a kernel arguments to
-disable Intel P-states. If your Linux bootloader is Grub, you can follow these
-steps:
+## Step 2 (Linux only): kernel and OS setup
 
-  * Edit /etc/default/grub (e.g. `sudo gedit /etc/default/grub`)
-  * Add `intel_pstate=disable` to `GRUB_CMDLINE_LINUX_DEFAULT`
-  * Run `sudo update-grub`
+The kernel must also be
```

Comment:
This is part of the krun-linux-kernel setup. You need to pull that section back up here I'm afraid.

Reply:
Huh?

Reply:
The tickless configuration is covered in the krun-linux-kernel docs.

Reply:
What does "pull ... back up" mean?

Reply:
I think I was suggesting a re-ordering, but now I'm not sure.

Basically the krun-linux-kernel repo documents how to make the kernel tickless.

I wrote a big comment on this, but I can't seem to find it...